### ISSUE TITLE
Notify popups - disabled pagination button styles

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-vanilla#350]
### Subject of the issue/enhancement/features
Pending the approval of https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/242, disabled pagination buttons in Hotgraphic popups will now be visible.  This leads to the question of what they should look like.  Currently, they will inherit disabled button styles which may or may not look out of place.

Thoughts on the below screenshots?  I prefer the lowered opacity look, but it may not be ideal for low vision users.

**Current view with disabled buttons hidden:**
![hotgraphic-disabled-button-hidden](https://user-images.githubusercontent.com/898168/185193881-0f5a4bb1-6a0f-4323-a6a8-d028c92fa2b7.jpg)

**After removing visibility: hidden**
![hotgraphic-disabled-button-visible](https://user-images.githubusercontent.com/898168/185193984-faaaccec-51a0-4d09-a173-982abdb7e4f0.jpg)

**Alternate style with lowered opacity**
![hotgraphic-disabled-button-translucent](https://user-images.githubusercontent.com/898168/185194061-937951fd-ac57-420a-bfe9-f119ec2450ff.jpg)

### Your environment
* Framework 5.x

**Comment by swashbuck** [comment]
@kirsty-hames @guywillis , any thoughts?

**Comment by oliverfoster** [comment]
I'd like them both to look more buttony in both enabled and disabled states.

**Comment by kirsty-hames** [comment]
> I'd like them both to look more buttony in both enabled and disabled states.

I agree. I think the default Adapt behaviour/look should always be what's most accessible and opacity wouldn't be ideal for low vision users. Having a consistent disabled state is good UX too.
